src/seamless_communication/cards/seamlessM4T_v2_large.yaml

```diff
@@ -7,8 +7,8 @@
 name: seamlessM4T_v2_large
 base: unity_nllb-100
 model_arch: base_v2
-char_tokenizer: "https://dl.fbaipublicfiles.com/seamless/models/spm_char_lang38_tc.model"
```

Comment:
I believe there are more such instances still remaining, could you search for "https://dl.fbaipublicfiles.com/seamless/models/spm_char_lang38_tc.model" and replace all instances with the new HF link?

Reply:
Perf! i covered all of them in the latest commit.